tests/test_filters.py

```diff
@@ -905,3 +905,35 @@ def test_poly_data_strip():
     slc = mesh.slice(normal='z', origin=(0,0,-10))
     stripped = slc.strip()
     assert stripped.n_cells == 1
+
+
+def test_shrink():
+
+    # mesh points
+    vertices = np.array([[0, 0, 0],
+                         [1, 0, 0],
+                         [1, 1, 0],
+                         [0, 1, 0],
+                         [0.5, 0.5, -1]])
+
+    # mesh faces
+    faces = [[4, 0, 1, 2, 3],  # square
+             [3, 0, 1, 4],     # triangle
+             [3, 1, 2, 4]]     # triangle
+
+    # number of points after shrink
+    n_points = 0
+    for face in faces:
+        n_points += face[0]
+
+    mesh = pyvista.PolyData(vertices, np.hstack(faces))
+    shrunk = mesh.shrink(shrink_factor=0.8)
+    assert shrunk.n_points == n_points
+    # assert shrunk.area < mesh.area
+
+
+    # Cube
+    mesh = pyvista.Cube()
+    shrunk = mesh.shrink(shrink_factor=0.8)
+    assert shrunk.n_points == mesh.n_points
+    # assert shrunk.area < mesh.area
```

Comment:
PyVista sent a message that shrunk is an `UnstructuredGrid` and therefore has no `area` property. Is this correct behavior?

Reply:
The `area` property is only valid for `PolyData`. The `vtkShrinkFilter` filter can be applied to any vtk dataset (not just poly data) hence why it always returns an `UnstructuredGrid`.

Reply:
Thanks @banesullivan . I could understood. So we cannot compare the area between orignail mesh and shrunk mesh?

Reply:
I understand :point_up: this question by :point_down: discussion.